features/cli/add_credential.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Storing AWS access credentials
+
+  In order to manage AWS access credentials
+  As an AWS IAM user
+  I want to store an AWS access credential
+
+  Scenario: Adding a long-term access credential without an MFA device
+    When I run `aws-session add credential` interactively
+    And I type "AKIAXXXXXXXXXXXX"
+    And I type "secretaccesskey1234567890"
+    And I type ""
+    And I type "default"
+    Then the exit status should be 0
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "MFA device (leave blank for none)"
+    And the stdout should contain "Credential alias"
+
+  Scenario: Adding a temporary security credential without an MFA device
+    When I run `aws-session add credential` interactively
+    And I type "ASIAXXXXXXXXXXXX"
+    And I type "secretaccesskey1234567890"
+    And I type "sessiontoken1234567890"
+    And I type ""
+    And I type "default"
+    Then the exit status should be 0
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "AWS Secret Access Key"
+    And the stdout should contain "AWS Session Token"
+    And the stdout should contain "MFA device (leave blank for none)"
+    And the stdout should contain "Credential alias"
+
+  Scenario: Adding a long-term access credential with a known MFA device
```

Comment:
This model has no concept of IAM users so an MFA device is optionally associated with a "credential."